### ISSUE TITLE
JSX indentation

### ISSUE THREAD

**Issue by LinusU** [feross/standard#358]
Currently it seems like standard doesn't lint the indentation on JSX files. For example the following file lints without issues:

```javascript
const React = require('react')

class Test extends React.Component {
  render () {
    return (
              <div>
       Test
          </div>
    )
  }
}

module.exports = Test
```

**Comment by Flet** [comment]
This should probably be moved to [eslint-config-standard-react](https://github.com/feross/eslint-config-standard-react).

Taking a look at it's [eslintrc.json](https://github.com/feross/eslint-config-standard-react/blob/master/eslintrc.json), I don't see the [jsx-indent-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent-props.md) rule enabled, which I think is what you're looking for.

**Comment by LinusU** [comment]
Hmm, it isn't `jsx-indent-props`, and as far as I can tell there isn't any plugin on that list that would do it. Maybe this should really be a job for the plugin that checks the indentation of normal code...

**Comment by feross** [comment]
There's currently a proposal for the rule you want: https://github.com/yannickcr/eslint-plugin-react/issues/342 Maybe you can weigh on that issue?

**Comment by LinusU** [comment]
Thanks!

**Comment by feross** [comment]
This is implemented in `eslint-plugin-react` now. Let's release it in v6.

**Comment by LinusU** [comment]
Awesome news! 🍾

**Comment by feross** [comment]
This is released in eslint-config-standard-react 2.0.0.
